### PR TITLE
Release 5.6

### PR DESCRIPTION
**Pull request by cverna**
Signed-off-by: Clement Verna <cverna@tutanota.com>

### CODE REVIEW

**Review by AdamSaleh — CHANGES_REQUESTED** [review]
We are hitting rate limits on docker when running integration tests. I'd rather we change these to i.e. quay?

File: bodhi/__init__.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """Fedora's update manager."""
 
-__version__ = "5.5"
+__version__ = "5.6"
```
Comment (mattiaverga) [inline bodhi/__init__.py]: Maybe this should set to `5.6.0` as Kevin had to patch the version number for packaging Bodhi in Koji

Comment (cverna) [reply bodhi/__init__.py]: :+1:

**Review by mattiaverga — APPROVED** [review]
LGTM

I think we can handle the integration tests fix post release. Postgres official docker image is only on dockerhub, I can't find any feasible replacement in quay (or fedora registry).  It will not be a quick fix I suppose.

**Pull request merged**